dev_guide/new_app.adoc

```diff
@@ -483,6 +483,37 @@ from the global library:
 ====
 
 image::console_select_image.png["Select Builder Image"]
+
+[NOTE]
+Only ImageStreamTags which have the 'builder' tag listed in their annotations will appear in this list, as demonstrated here:
+
+----
+    {
+      "kind": "ImageStream",
+      "apiVersion": "v1beta3",
+      "metadata": {
+        "name": "ruby",
+        "creationTimestamp": null
+      },
+      "spec": {
+        "dockerImageRepository": "registry.access.redhat.com/openshift3/ruby-20-rhel7",
+        "tags": [
+          {
+            "name": "2.0",
+            "annotations": {
+              "description": "Build and run Ruby 2.0 applications",
+              "iconClass": "icon-ruby",
```

Comment:
no...it's barely even an appropriate place to explain the builder tag.  all this really belongs in explicit documentation of imagestreams, but the only place imagestreams are talked about is in the "architecture" section and I did not want to compound that situation.

Basically we need an imagestream doc in the developer or admin section that actually explains how to create/manage/author imagestreams, and then this builder tag should be documented there (along with icons, etc).

but i'm not signing up for that at the moment.